index.js

```diff
@@ -0,0 +1,54 @@
+webix.ui(
+    {rows: [
+            {view: "toolbar", css: "webix_dark", cols: [
+                    {view: "label", label: "My App"},
+                    {},
+                    {view: "button", type: "icon", icon: "wxi-user", label: "Profile", width: 100, css: "webix_transparent"}
+                ]
+            },
+            {cols: [
+                    {type: "clean", css: "navigation-background", rows: [
+                            {view: "list", template: "#title#", select: true, css: "navigation-background", data: [
```

Comment:
According to the task, List should not contain a vertical scroll.
Also, Datatable should not have a horizontal scroll.